job-search-react/src/components/JobApp/Card/Card.js

```diff
@@ -1,6 +1,8 @@
 import React from 'react';
 
 export default function Card({ isSubmitted, jobData }) {
+	let date = new Date(jobData.created);
```

Comment:
Do we need to worry if `jobData.created` or `jobData.title` don't exist? Would the card ever get called with those properties not on `jobData`?

Reply:
We added a try catch